adal/src/telemetry/java/com/microsoft/aad/adal/BrokerEvent.java

```diff
@@ -57,6 +57,32 @@ void setBrokerAccountServiceConnected() {
         setProperty(EventStrings.BROKER_ACCOUNT_SERVICE_CONNECTED, Boolean.toString(true));
     }
 
+    void setServerErrorCode(final String errorCode) {
+        if (null != errorCode && !errorCode.equals("0")) {
```

Comment:
Why the errorCode checking (null != errorCode) here is not the same in HttpEvent.java (!StringExtensions.isNullOrBlank(errorCode))?

Reply:
Good catch. I'll make these the same 👍

Reply:
Fixed in d2dd52acf54853cb395b40375135d8aebcc28741